tasks/mx/inegi.py

```diff
@@ -253,12 +262,17 @@ def columns(self):
 
     def populate(self):
         session = current_session()
-        session.execute('INSERT INTO {output} (the_geom, cvegeo) '
-                        'SELECT wkb_geometry, cvegeo '
+        column_targets = self.columns()
+        output_cols = ', '.join(column_targets.keys())
+        input_cols = ','.join(['{}::{}'.format(colname, ct.get(session).type)
+                               for colname, ct in column_targets.iteritems()])
+        session.execute('INSERT INTO {output} ({output_cols}) '
+                        'SELECT {input_cols} '
```

Comment:
`{input_cols}` here is actually pulling from the same definition as output columns -- so this doesn't work.  For example, on the output, the geometry is called `the_geom`, but on input it's called `wkb_geometry`.  This needs to be adjusted to match column names from the input table.